topology/graph/memory.go

```diff
@@ -88,11 +88,18 @@ func (m *MemoryBackend) GetEdgeNodes(e *Edge, t GraphContext, parentMetadata, ch
 		child = n
 	}
 
-	if parent == nil || child == nil {
-		return nil, nil
+	pn := []*Node{}
```

Comment:
This is done on purpose : if an edge has no source or destination, it is invalid.
What problem does it cause ?

Reply:
```BothV().Has()``` failing because of this

Reply:
Fixed this in BothV method.
Thanks!